packages/react-charting/src/components/Legends/Legends.base.tsx

```diff
@@ -103,7 +103,7 @@ export class LegendsBase extends React.Component<ILegendsProps, ILegendState> {
         hoverAction: legend.hoverAction!,
         onMouseOutAction: legend.onMouseOutAction!,
         color: legend.color,
-        shape: legend.shape,
+        shape: shape,
```

Comment:
you will need to generate a change file by running `yarn change` from the project root